README.md

```diff
@@ -43,23 +48,48 @@ port          | 8080                               | Port-number to run the Hood
 bindAddress   | 127.0.0.1                          | Address that Hoodie binds to
 public        | path.join(options.path, 'public')  | path to static assets
 inMemory      | false                              | Whether to start the PouchDB Server in memory
-dbUrl         | PouchDB Server                     | If provided does not start PouchDB Server and uses external CouchDB. Has to contain credentials.
+dbUrl         | –                                  | If provided uses external CouchDB. Has to contain credentials.
 data          | path.join(options.path, '.hoodie') | Data path
 
-If that doesn’t make much sense just yet, don’t worry about it.
+Hoodie is using the [rc](https://www.npmjs.com/package/rc) module to retrieve
+configuration from CLI arguments, environment variables and configuration files.
 
 ## Testing
 
-The `@hoodie/server` test suite is run with `npm test`.
+Local setup
+
+```
+git clone https://github.com/hoodiehq/hoodie.git
+cd hoodie
+npm install
+```
+
+The `hoodie` test suite is run with `npm test`.
+
+You can [read more about testing Hoodie](test)
+
+## Architecture
+
+Hoodie is server built on top of [hapi](http://hapijs.com) with frontend APIs
+for account and store related tasks.
 
-The tests live in `test/unit` and `test/integration`. `test/unit` tests (or “unit tests”) are to test the behaviour of individual sub-modules within `@hoodie/server`, while `test/integration` tests (or “integration tests”) are used to test the behaviour of a fully running instance of `@hoodie/server`, e.g. the behaviour of its HTTP API.
+It consists of three main components
 
-If you are adding new features to `@hoodie/server` you should provide test cases for the new feature. Depending on the feature, it's either best to write unit tests or integration tests and sometimes even both. The more tests we have, the more confidently we can release future versions of `@hoodie/server`.
+1. [**account**](https://github.com/hoodiehq/hoodie-account)  
+   Hoodie’s account module. It exposes a [JSON API](http://jsonapi.org/),
+   a corresponding client and a generic account UI.
```

Comment:
Does the `account` module expose a `UI` or `API`? 

Is it more accurate to say "It exposes a set of JSON API endpoints, as well as a corresponding  client and generic account API?